humble/distribution.yaml

```diff
@@ -1359,6 +1359,16 @@ repositories:
       url: https://github.com/introlab/find-object.git
       version: humble-devel
     status: maintained
+  fluent_bit_vendor:
+    release:
+      tags:
+        release: release/humble/{package}/{version}
+      url: https://github.com/minipada/fluent_bit_vendor-release.git
```

Comment:
My initial guess is a problem with the release repository it doesn't have the release/humble/{package}/{version} branch. I see you are trying to create a release in ros2-gbp, so first just add the source entry here, then continue the process in ros2-gbp, then you can create another PR to add the release repository. More details here https://docs.ros.org/en/rolling/How-To-Guides/Releasing/Release-Team-Repository.html#what-if-my-existing-release-repo-isn-t-on-ros2-gbp.
Also please consider following the recommended release workflow using Bloom https://docs.ros.org/en/rolling/How-To-Guides/Releasing/First-Time-Release.html.